### ISSUE TITLE
Can't access 'cookies' inside command call block

### ISSUE THREAD

**Issue by grig191** [andypike/rectify#20]
When calling command like this:

```ruby
SomeCommand.call(params) do
  on(:some_message) do
     cookies[:random_cookie] = 'random string'
  end
end
```
will throw:

`undefined local variable or method 'cookies' for #<SomeCommand:>`

So, if I want to use cookies inside this block, I write some unusual code:

```ruby
cookiess = cookies
SomeCommand.call(params) do
  on(:some_event) do
     cookiess[:random_cookie] = 'random string'
  end
end
```

Above code works fine.
Dev env:

> ruby 2.2.1p85 (2015-02-26 revision 49769) [x86_64-linux]

> Rails 4.2.5.2

> Rectify 0.6.0

**Comment by andypike** [comment]
Thanks for the report @grig191 💖 

I'll take a look and see what I can do to make this work in a nicer way.

**Comment by grig191** [comment]
I suggest it is not working because `cookies` is not variable, it is private method

**Comment by grig191** [comment]
Adding below code to Rectify::Command class fixes it,

```ruby
    def cookies
      @@controller.method(:cookies).call
    end

    def self.controller_class(controller_class)
      a = ObjectSpace.each_object(controller_class).to_a
      @@controller = a.last
    end
```

But there is another problem: you need to define controller class like this:

```ruby
class SomeCommand < Rectify::Command
  controller_class SomeController

  #...
```

I don't think that it is perfect solution, but it fixs this issue

**Comment by andypike** [comment]
Ah yes, we have this issue already here: https://github.com/andypike/rectify/issues/15. I'll see if there is a solution that means we can use all private methods. I'll take and will let you know how I get on, thanks for the information, that's very useful 💖

**Comment by andypike** [comment]
@grig191 I've pushed a fix up for this issue (see 2adf21ed6078b5552b5678716d6a185f15ad5128). Could you update your Gemfile to try this one out with:

```
gem "rectify", :github => "andypike/rectify"
```

If this works for you then I'll release a new version to RubyGems.

Thanks! 💖

**Comment by grig191** [comment]
Now it works perfectly! Thank you Andy!

**Comment by andypike** [comment]
Thanks @grig191 that's great. I've released v0.6.1 that contains this fix.

Thanks again for the issue report!
